src/CLI/Check.php

```diff
@@ -72,8 +86,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
             $this->readRules($config, $rulesFilename);
 
-            $runner = new Runner();
-            $runner->run($config, $progress, $targetPhpVersion);
+            $runner = new Runner($stopOnFailure);
+            try {
+                $runner->run($config, $progress, $targetPhpVersion);
+            } catch (FailOnFirstViolationException $e) {
```

Comment:
this empty catch is the worst part in my opinion...

Reply:
Uhm, yes but this solution is much better and clean IMHO.